nemo_rl/distributed/worker_groups.py

```diff
@@ -272,6 +298,27 @@ def __call__(
         worker._RAY_INITIALIZER_ACTOR_REF_TO_AVOID_GC = isolated_initializer
         return worker
 
+    def get_all_worker_results(
```

Comment:
why is this function necessary? Can't we call get_results on the future (even with the proxy arg?)

Reply:
Was redundant, removed it in the latest iteration.